doc(release): Prepare release notes for collect 22.10.1 (#2187)
centreon/centreon-documentation: i18n/fr/docusaurus-plugin-content-docs/version-22.10/releases/centreon-os.md

## Centreon Collect

### 22.10.1

Release date: `March 15, 2023`

#### Centreon Broker

##### Bug fix

> - Fixed a defect in the Stream Connector mechanism that would not handle the BBDO v3 RRD rebuild events.

Reviewer: ```suggestion
- Fixed an issue in the Stream Connector mechanism that would not handle the BBDO v3 RRD rebuild events.
```
